plugins/modules/zabbix_discovery_rule.py

```diff
@@ -0,0 +1,700 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2013-2014, Epic Games, Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
```

Comment:
Can you please remove `ANSIBLE_METADATA`?  
`ANSIBLE_METADATA` is no longer necessary.

see:
https://github.com/ansible-collections/overview/issues/57
https://github.com/ansible-collections/community.zabbix/pull/48